crates/subspace-node/src/service/overseer.rs

```diff
@@ -1,189 +0,0 @@
-// Copyright 2017-2020 Parity Technologies (UK) Ltd.
```

Comment:
I have no problem with removing the `overseer.rs`, the main reason I kept it is to minimize the diff from the upstream, but it's fine to refactor it this way.